helper/bearer.go

```diff
@@ -25,9 +25,28 @@ import (
 	"strings"
 )
 
-func BearerTokenFromRequest(r *http.Request) string {
-	auth := r.Header.Get("Authorization")
-	split := strings.SplitN(auth, " ", 2)
+const (
+	defaultAuthorizationHeader = "Authorization"
+)
+
+type BearerTokenLocation struct {
+	Header         *string `json:"header"`
+	QueryParameter *string `json:"query_parameter"`
+}
+
+func BearerTokenFromRequest(r *http.Request, tokenLocation *BearerTokenLocation) string {
+	var token string
+	if tokenLocation != nil {
+		if tokenLocation.Header != nil {
+			token = r.Header.Get(*tokenLocation.Header)
```

Comment:
Wouldn't it make more sense to return the value here directly?

```suggestion
			return r.Header.Get(*tokenLocation.Header)
```